transport_nantes/authentication/templates/registration/password_reset_form.html

```diff
@@ -0,0 +1,45 @@
+{% extends 'asso_tn/base_transport_nantes.html' %}
+{% load static %}
+
+
+{% block app_content %}
+
+<style type="text/css">
```

Comment:
These changes deserve separate comment.  Why do they belong here and not in `transport_nantes.css`, for example?  Why do you provide a maximum width of 330 pixels on the sign-in form?  Does that work in a responsive context (on different screen sizes)?